ignore hidden or swap files in templates

The following error occured if i run the binary while vim places temporary swap files (like `.dashboard.html.swp`) in the `templates` directory.

```
$./kasse -hardware=false -listen=0.0.0.0:9000
panic: template: content:10: unexpected {{end}}

goroutine 1 [running]:
html/template.Must(0x0, 0x7f06a95e3b00, 0xc20802cca0, 0x0)
	/usr/lib/go/src/html/template/template.go:304 +0x50
main.init·2()
	/home/johannes/stuff/go/src/github.com/nnev/kasse/templates.go:50 +0x81d
main.init()
	/home/johannes/stuff/go/src/github.com/nnev/kasse/templates.go:59 +0x601

goroutine 8 [runnable]:
text/template/parse.lexRightDelim(0xc208062280, 0x9c02c8)
	/usr/lib/go/src/text/template/parse/lex.go:263 +0xb7
text/template/parse.(*lexer).run(0xc208062280)
	/usr/lib/go/src/text/template/parse/lex.go:198 +0x5d
created by text/template/parse.lex
	/usr/lib/go/src/text/template/parse/lex.go:191 +0x1ac
```

--- a/templates.go
+++ b/templates.go
@@ -34,6 +34,10 @@ func init() {
 		if filepath.Base(f) == "layout.html" {
 			continue
 		}
+		// Skip hidden files
+		if filepath.Base(f)[0] == '.' {
+			continue
+		}
 		content, err := ioutil.ReadFile(f)
 		if err != nil {
 			log.Fatalf("Could not read %q: %v", f, err)

